README.md

```diff
@@ -395,6 +395,11 @@ try {
 
 ## Error messages
 
+Values are in the `code` field of the rejected Error object.
+
+- OAuth Authorization [error codes](https://tools.ietf.org/html/rfc6749#section-4.1.2.1)
```

Comment:
**Note!** Adding more possible error codes here might be considered a breaking change according some conventions. For example someone expecting some errors to always include `code: 'token_refresh_failed'` might be disappointed when discovering that a new version of the library replaced the value with `code: 'invalid_grant'` in some situations.

If we don't want to use the same `code` field for the new values, we could move them to a separate field such as `oauthCode`. This would require some extra work though to move the value through the promise rejection call.

Reply:
Yeah it's definitely a breaking change currently. As tempting as it is to move the auth code to an `oauthCode` field, I think the "right" thing to do is to surface the actual error codes and do a major release for it.